src/api/app/controllers/source_controller.rb

```diff
@@ -1042,6 +1041,8 @@ def package_command_waitservice
 
   # POST /source/<project>/<package>?cmd=mergeservice
   def package_command_mergeservice
+    verify_can_modify_target!
```

Comment:
This already happens in `package_command` through `validate_target_for_package_command_exists!`